examples/baselines/act/act/detr/backbone.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+"""
+Backbone modules.
+"""
+from collections import OrderedDict
+
+import torch
+import torch.nn.functional as F
+import torchvision
+from torch import nn
+from torchvision.models._utils import IntermediateLayerGetter
+from typing import Dict, List
+
+from ..utils import NestedTensor, is_main_process
```

Comment:
use absolute imports when possible, it is just the style choice this repo uses.

Reply:
I changed them to absolute imports, but I'm not sure if they are correct. please let me know if they need to be fixed. I also added a README file.